app/views/events/index.html.erb

```diff
@@ -25,6 +25,12 @@
         <td><%= link_to 'Show', event %></td>
         <td><%= link_to 'Edit', edit_event_path(event) %></td>
         <td><%= link_to 'Attend', attend_path(event) %></td>
+        <% if event.creator_id == @current_user.id %>
+          <td><%= button_to 'Destroy', event, method: :delete,data: { confirm: 'Are you sure?' }, class: 'btn btn-danger'%></td>
+        <% else %>
+          <td></td>
+        <% end %>
```

Comment:
This code is not for this PR, why is here? Remember the idea of create branches is to avoid this. I noticed you merged BEFORE making the PR, that is the problem, you should make the PR from the branch in your fork

Reply:
it's my mistake because I didn't do PR before, for the rest, I'll wait until it's merger before continuing I think.